### DISCUSSION TITLE
error: "window is not defined" in Next.js 13 with react-auth-kit

### DISCUSSION THREAD

**Discussion by leodevbro** [react-auth-kit/react-auth-kit#1227] [Q&A]
Please help, how can I setup react-auth-kit in Next.js project?

I don't use `window` anywhere in the project, but I guess the react-auth-kit library is trying to use it somehow.

![image](https://user-images.githubusercontent.com/54504880/224968364-4a56e939-74c2-47a4-b3a8-2a4793a7cb83.png)

**Comment by DamienA24** [comment]
Hello, I have same problem... any solutions ? thanks

**Reply by leodevbro** [reply]
I think this is the solution for Next.js but I have not tested it.
https://github.com/mac-s-g/react-json-view/issues/425
